.github/scripts/install_olm.sh

```diff
@@ -0,0 +1,15 @@
+if [ ! -f "./install-olm.sh" ]; then
+    curl -L https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.21.2/install.sh -o install.sh
```

Comment:
As far as I can see we are using(at the moment) OLM only for Keycloak.
The Keycloak operator has been developed to favor "production-ready" environments and it might be a little trickier to maintain for development purposes.
I would suggest switching to a straight `Deployment`, e.g.:
https://github.com/bf2fc6cc711aee1a0c2a/srs-fleet-manager/blob/main/dist/k8s-dev/keycloak.yaml
to be more easily maintainable, and avoid the installation of OLM.
wdyt?

Reply:
Hi, I think exactly otherwise. With just deployment, we still have to maintain the correct Deployment version, this way we just take care of the subscription version. And also we should try apicurio operator so we will need OLM anyways.

Reply:
To be exact we have to try bundle and olm of apicurio operator.

Reply:
Anyway, I will remove this part from this PR as we have issue for kubernetes runs open we will figure it out later.

Reply:
Sorry for the delay answering here @obabec !

> With just deployment, we still have to maintain the correct Deployment file, this way we just take care of the subscription version.

This is not entirely correct, in the sense that you need to maintain, `OperatorGroup`, `Subscription`, and 2 additional (and not stable) CRs.

Apart from this note, if you are going to use OLM for everything I have no strong arguments against it as the infrastructure is justified to mimic more closely a "real environment" like OCP. 👍